feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt

```diff
@@ -7,35 +7,57 @@ import androidx.compose.runtime.*
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.stringArrayResource
 import androidx.compose.ui.res.stringResource
-import androidx.core.text.isDigitsOnly
+import androidx.hilt.navigation.compose.hiltViewModel
 import com.iteneum.designsystem.components.*
 import com.iteneum.designsystem.theme.LPTypography
 import com.iteneum.designsystem.theme.LeasePertTheme
 import com.iteneum.repair.R
+import com.iteneum.repair.data.RepairViewModel
 
 /**
  * Function that creates [RepairView] compose. for user's service request.
  * User will require to fill all the correspondent fields on the screen.
  *
+ * Function [RepairView] has
+ * @param navigateToApartment - to get the function to navigate ApartmentView when clicking button
+ * @param repairViewModel - to get the Repair View Model access
+ *
+ * Function [RepairContainer] has
+ * @param unitDepartment - to use if for displaying Unit in the Unit Text Field
+ * @param repairViewModel -  to get the Repair View Model access
+ * @param navigateToApartment - its passed from RepairView() for same purpose
+ *
  * @author Jose Miguel Garcia Reyes
  */
 
+@Composable
+fun RepairView(
+    navigateToApartment: () -> Unit,
+    repairViewModel: RepairViewModel = hiltViewModel()
+) {
+    LaunchedEffect(true) {
+        repairViewModel.getInformation()
+    }
+    RepairContainer(
+        unitDepartment = repairViewModel.repairModelRead.value.unitDepartment,
+        navigateToApartment = navigateToApartment
+    )
+}
+
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun RepairView() {
+fun RepairContainer(
+    unitDepartment: String,
+    repairViewModel: RepairViewModel = hiltViewModel(),
```

Comment:
We don't need send the viewModel to a second composable view, viewModel always must be in the first one composable view, if you want to send a value or an object to the second one you need to access to that value and then add a new param to the second composable function.

Reply:
Ok, modified! Aided by you, thank you very much!